practice-one/pages/index.html

```diff
@@ -166,7 +166,7 @@ <h2 class="section-title">Grow Your Business With Our Agency</h2>
             <li>Web and email hosting server</li>
           </ul>
         </div>
-        <img src="../assets/images/Frame 20.png" alt="About Us">
+        <img src="../assets/images/Frame20.png" alt="About Us">
```

Comment:
Shouldn't use numbers when naming and no need to uppercase

Reply:
Thank you for your comments. I will fix it